README.md

```diff
@@ -38,6 +38,11 @@ For training, we use [pytorch-lightning](https://www.pytorchlightning.ai/index.h
     * The following features are now independent: weighting of the diffusion loss function (`sgm/modules/diffusionmodules/denoiser_weighting.py`), preconditioning of the network (`sgm/modules/diffusionmodules/denoiser_scaling.py`), and sampling of noise levels during training (`sgm/modules/diffusionmodules/sigma_sampling.py`).
 - Autoencoding models have also been cleaned up.
 
+### Style
+
+* The repo is formatted with [Black](https://github.com/psf/black) and linted with [Ruff](https://beta.ruff.rs/).
+  * You can easily have these run on every commit by installing [`pre-commit`](https://pre-commit.com/) (`pip install pre-commit`) and running `pre-commit install` in the repo root.
```

Comment:
Remove space before *

Reply:
Why? It's meant to be a sub-point.

![Screenshot 2023-07-25 at 14 30 47](https://github.com/Stability-AI/generative-models/assets/58669/0bb9b683-4308-4406-946d-1c995f508705)